opendcs-web-client-jetty/src/main/java/org/opendcs/odcswebclient/jetty/Start.java

```diff
@@ -37,30 +38,57 @@ public class Start
     public static void main(String[] args) 
             throws Exception
     {
+        Log.setLog(new StdErrLog());
+
         String warFilePath = null;
         int port = 8080;
         String contextPath = "/";
+        String apiFileDetailsPath = null;
         // Initialize the JETTY server and servlet holders.
         //Server server = new Server(apiCmdLineArgs.getPort());
         ArgParser argParse = new ArgParser(args);
-        String portString = argParse.switchValue("-p", "8080");
+        String portString = argParse.switchValue("-p", "8083");
         System.out.println("Using Port: " + portString);
-        try {
+        try
+        {
             warFilePath = argParse.switchValue("-w", null);
         }
         catch (Exception e)
         {
             System.out.println("War File Not Provided!");
             System.exit(1);
         }
+
+        try
+        {
+            apiFileDetailsPath = argParse.switchValue("-f",  null);
+        }
+        catch (Exception e)
+        {
+            System.out.println("No Api File Details Path File Provided.  Using defaults.");
```

Comment:
For the system.outs in this file, a logger will be added, but that will be in a separate pull request.  the Start.java file in opendcs-rest-api-jetty is using a logger and I want to make sure that looks good before creating the new pull request for the rest of the project.